yarn-project/end-to-end/src/e2e_l1_to_l2_msg.test.ts

```diff
@@ -175,4 +183,56 @@ describe('e2e_l1_to_l2_msg', () => {
     expect(consumptionReceipt.status).toBe(TxStatus.MINED);
     await expectBalance(ownerAddress, mintAmount + initialBalance - transferAmount);
   }, 80_000);
+
+  it('cancelled l1 to l2 messages cannot be consumed', async () => {
+    const initialBalance = 10n;
+    const [ownerAddress] = accounts;
+    const ownerPub = await aztecRpcServer.getAccountPublicKey(ownerAddress);
+    const deployedL2Contract = await deployContract(initialBalance, pointToPublicKey(ownerPub));
+    await expectBalance(accounts[0], initialBalance);
+
+    const l2TokenAddress = deployedL2Contract.address.toString() as `0x${string}`;
+
+    const { secretString, deadline } = await sendL1ToL2MessageViaPortal(ownerAddress, l2TokenAddress);
+    // Wait 5s for the archiver to process the message
+    await delay(5000);
+
+    // set the block timestamp to be after the deadline (so we can cancel the message)
+    const params = `["${deadline + 1}"]`;
+    await fetch(config.rpcUrl, {
```

Comment:
We might want to create a small e2e util that has this and other evm manipulation functions that could be handy.

Reply:
easier to create a utils.ts and shove everything including `setup` there!